.env.test

```diff
@@ -71,3 +71,6 @@ CORS_ALLOW_ORIGIN='^https?://localhost(:[0-9]+)?$'
 # Choose one of the transports below
 MESSENGER_TRANSPORT_DSN=in-memory://
 ###< symfony/messenger ###
+
+WEBHOOK_USERNAME=testUsername
+WEBHOOK_PASSWORD=testPassword
```

Comment:
Same proposal about changing naming to `BASIC_AUTH_USERNAME` / `BASIC_AUTH_PASSWORD`

Reply:
Agreed and renamed